### ISSUE TITLE
Change prog parameter for argparse subcommands

### ISSUE THREAD

**Issue by luizirber** [dib-lab/sourmash#415]
Punting from https://github.com/dib-lab/sourmash/pull/390#discussion-diff-170345804R18

Change ArgumentParser `prog` parameter to account for proper subcommand calling:
```
    p = argparse.ArgumentParser(prog="sourmash lca compare_csv") 
```
Without this the help messages are a bit wonky.

**Comment by ctb** [comment]
Ref #785, and maybe #931.

**Comment by ctb** [comment]
I think this was fixed in #785.
